extensions/flac/src/main/jni/flac_jni.cc

```diff
@@ -117,21 +117,61 @@ DECODER_FUNC(jobject, flacDecodeMetadata, jlong jContext) {
     }
   }
 
+  jobject pictureList = env->NewObject(arrayListClass, arrayListConstructor);
```

Comment:
pictures

Reply:
There's already a pictures variable, so renaming this to jPictures.